db/migrate/20160907183911_add_author_id_to_books.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorIdToBooks < ActiveRecord::Migration
+  def change
+    add_column :books, :author_id, :integer
```

Comment:
Probably an index by `author_id` is necessary. You can do that in two different ways.

`add_references :books, :author, index: true`

or

```
add_column :books, :author_id, :integer
add_index    :books, author_id
```

I prefer the first one